javalin/src/test/java/io/javalin/TestRequest.kt

```diff
@@ -7,6 +7,9 @@
 package io.javalin
 
 import io.javalin.http.Header
+import io.javalin.http.formParamAsClass
```

Comment:
Why do these have to be imported now?

Reply:
Extension functions have to be imported in Kotlin - and all variants of inline functions are now extension functions due to limitation of interfaces (in theory it makes sense, so Java users are not forced to implement this). It doesn't change anything despite that, because it's still available in IDE hints etc.